change/@azure-msal-browser-09b253d8-3271-4ccb-aa4c-834e213cbfe2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "tokenQueryParameters are now sent to the /token endpoint for all Confidential Client flows. They're also explicitly excluded from Public Client flows that don't use them. #5573",
```

Comment:
Please update this comment to be more descriptive of what changed in msal-browser (CCA isn't relevant to browser so should not be mentioned)